examples/dune

```diff
@@ -9,36 +9,6 @@
  (libraries graphics))
 
 (executable
- (name concentric_circles)
```

Comment:
Why are these removed from the dune file? Could you please check if they're in sync with the latest `main` branch.

Reply:
Ok @Sudha247